src/stores/exitHandler.ts

```diff
@@ -18,9 +17,14 @@ import Transactions from '../components/txNotification/transactions';
 import { InflightTxReceipt } from '../utils/types';
 import Web3Store from './web3/';
 import PlasmaConfig from './plasmaConfig';
+import { Outpoint, Unspent } from 'leap-core';
+
 
 export default class ExitHandler extends ContractStore {
 
+  @observable 
```

Comment:
This has to go, not being used and no definition of ExitData breaks build.

Reply:
fixed.